dev_scripts/install

```diff
@@ -33,6 +33,9 @@ make_sandbox() {
 	# to _build/runtime_params and be used for munin module installation.
 	./Build --install_base "${BASEDIR}/sandbox" install
 
+	# disable taint mode in sandbox
```

Comment:
Could you explain (and maybe comment in the code), why the taint mode should be disabled in the sandbox environment? Wouldn't this be a good place to be informed about tainting issues early in development?